src/Diagnostics.ModelsAndUtils/Models/ResponseExtensions/Dropdown.cs

```diff
@@ -57,17 +70,53 @@ public static class ResponseDropdownExtension
         ///     data.Add(new Tuple<![CDATA[<string, bool, Response>]]>(firstDataKey, selected, firstDataEntry));
         ///
         ///     Dropdown dropdownViewModel = new Dropdown(label, data);
-        ///     res.AddDropdownView(dropdownViewModel);
+        ///     res.AddDropdownView(dropdownViewModel,"Title");
         /// }
         /// </code>
         /// </example>
         public static DiagnosticData AddDropdownView(this Response response, Dropdown dropdownView, string title = null)
+        {
+            return AddDropdownView(response, dropdownView, title, DropdownType.Legacy,DropdownPosition.FloatRight);
+        }
+
+        /// <summary>
+        /// Adds a Dropdown View to Response
+        /// </summary>
+        /// <param name="response">Response</param>
+        /// <param name="dropdownView">Dropdown View</param>
+        /// <param name="title">Title</param>
+        /// <param name="type">Dropdown Type</param>
+        /// <param name="position">Dropdown Position</param>
+        /// <returns></returns>
+        /// <example>
+        /// This sample shows how to use <see cref="AddDropdownView"/> method.
+        /// <code>
+        /// public async static Task<![CDATA[<Response>]]> Run(DataProviders dp, OperationContext cxt, Response res)
+        /// {
+        ///      string label = "select item here";
+        ///      List<![CDATA[<Tuple<string, bool, Response>>]]> data = new List<![CDATA[<Tuple<string, bool, Response>>]]>();
+        ///
+        ///      string firstDataKey = "key1";
+        ///      bool selected = true;
+        ///      var firstDataEntry = new Response();
+        ///      firstDataEntry.AddMarkdownView(@"some markdown content");
+        ///
+        ///     data.Add(new Tuple<![CDATA[<string, bool, Response>]]>(firstDataKey, selected, firstDataEntry));
+        ///
+        ///     Dropdown dropdownViewModel = new Dropdown("label", data);
+        ///     res.AddDropdownView(dropdownViewModel,"Title",DropdownType.Fabric,DropdownPosition.FloatLeft);
+        /// }
+        /// </code>
+        /// </example>
+        public static DiagnosticData AddDropdownView(this Response response, Dropdown dropdownView, string title, DropdownType type = DropdownType.Legacy,DropdownPosition position = DropdownPosition.FloatRight)
```

Comment:
For this one, does it make sense to not default the position and let user specify it. If you want to default it, having left position be default does make more sense..